docs/faq.md

```diff
@@ -1,3 +1,24 @@
+## How can I speed up my build?
```

Comment:
I worry this title is too general - how about something like "How do I avoid running builders for unnecessary inputs?"